draftivist_ui/src/components/DraftClosing.tsx

```diff
@@ -0,0 +1,66 @@
+import * as m from "mithril";
+import { Campaign, Issue } from "../models"
+import { BaseComponent, Link, elementAttrs } from "./base"
+import { PreviewDraftCTA } from './PreviewDraftCTA';
+
+type Attrs = {
+  campaign: Campaign
+  issue: Issue
+}
+
+export default function (): BaseComponent<Attrs> {
+  // add a function to save the intro
+  // add a function to save the conclusion
+
+  function saveDraft(campaign: Campaign, content: string) {
+    console.log("running conclusion")
```

Comment:
we should either remove this or update it to have some more specific log information if you'd like to keep it around while we work through connecting the backend and frontend apps.